nano/templates/song.tmpl

```diff
@@ -0,0 +1,87 @@
+<!--
+Title: Instrument UI
+Used In File(s): \code\game\objects\structures\musician.dm
+-->
+
+<div class="block">
+	<div class="item">
+		<div class="itemLabel"><b>Playback</b></div>
+		<div class="itemContent">
+			{{:helper.link('New', 'document', {'newsong': 1})}}
+			{{:helper.link('Import', 'folder-open', {'import': 1})}}
+			{{:helper.link('Play', 'play', {'play': 1}, data.lines.length > 0 ? (data.playing ? 'selected' : null) : 'disabled')}}
+			{{:helper.link('Stop', 'stop', {'stop': 1}, !data.playing ? 'selected' : null)}}
+		</div>
+	</div>
+	<div class="item">
+		<div class="itemLabel">Repeat Song:</div>		
+		{{:helper.link('-', null, {'repeat' : -10}, data.repeat > 0 && !data.playing ? null : 'disabled')}}
+		{{:helper.link('-', null, {'repeat' : -1}, data.repeat > 0 && !data.playing ? null : 'disabled')}}
+		<span style="float: left; width: 80px; text-align: center;">{{:data.repeat}}</span>
+		{{:helper.link('+', null, {'repeat' : 1}, data.repeat < data.maxRepeat && !data.playing ? null : 'disabled')}}
+		{{:helper.link('+', null, {'repeat' : 10}, data.repeat < data.maxRepeat && !data.playing ? null : 'disabled')}}
+	</div>
+	<div class="item">
+		<div class="itemLabel">Tempo:</div>		
+		{{:helper.link('-', null, {'tempo' : 10}, data.tempo < data.maxTempo ? null : 'disabled')}}
+		{{:helper.link('-', null, {'tempo' : 1}, data.tempo < data.maxTempo ? null : 'disabled')}}
+		<span style="float: left; width: 80px; text-align: center;">{{:helper.round(600 / data.tempo)}} BPM</span>
+		{{:helper.link('+', null, {'tempo' : -1}, data.tempo > data.minTempo ? null : 'disabled')}}
+		{{:helper.link('+', null, {'tempo' : -10}, data.tempo > data.minTempo ? null : 'disabled')}}
+	</div>
+</div>
+
+<h3>Editor</h3>
+{{for data.lines}}
+	<div class="line" style="padding:4px;{{if index % 2 == 1}}background-color: #444{{/if}}">
```

Comment:
I would have used striped, but that is unfortunately a class you apply to a table, and then it directly references the tr items in it. (ie `.striped tr:nth-child(even)` ). I don't remember if there's a way to make that work in a general way and also keep it as a class for the parent.